### DISCUSSION TITLE
How can I make it so I have to press Tab before I can expand a Snippet with Enter?

### DISCUSSION THREAD

**Discussion by mizuguru** [L3MON4D3/LuaSnip#950] [Q&A]
Hi. First of, sorry if the question is not clear enough, I just don't know how to phrase it.

I want to use LuaSnip to write LaTeX, and everything is working perfectly for me, asides from this issue that I have.

Let's say I've just finished writing a paragraph and I want to start a new one. I press the Enter key to make a new line below, but since the paragraph ends with `.`, it transforms that dot into `\dots` (`...`) instead of adding a new line.

To solve this problem, I would like to make it so it is required to press Tab (which is the key I have defined to jump between snippets and expand them) before I can actually select a Snippet and then press the Enter key to expand it.

Is there any way to do this?

I'm sorry if this is a noob question (it's because I am one D:)

**Comment by L3MON4D3** [comment]
I'm a bit confused, why is a snippet expanded on enter, when you have the mapping (presumably) only set for tab? Has it something to do with nvim-cmp-autocompletion?
